src/database/DataTypes/ItemBatch.js

```diff
@@ -21,6 +21,10 @@ import Realm from 'realm';
  * @property  {List.<TransactionBatch>}  transactionBatches
  */
 export class ItemBatch extends Realm.Object {
+  get otherPartyName() {
+    return `${this.supplier?.name || ''}`;
```

Comment:
Is the template string needed here?

Reply:
nope 1133d8b9b950e87191337874fe393ffa7efad04f